shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -47,8 +47,13 @@ extern NSNotificationName const FlutterSemanticsUpdateNotification;
  * Dart-related state and asynchronous tasks when navigating back and forth between a
  * FlutterViewController and other `UIViewController`s.
  */
+#ifdef __IPHONE_13_4
```

Comment:
You shouldn't need this, the presence of `UIPointerInteractionDelegate` will exist depending on the SDK used for compilation.  We don't need to support old versions of Xcode.